src/locales/fr/translation.json

```diff
@@ -41,14 +41,18 @@
       "selectWallets": "Sélectionnez les portefeuilles à ouvrir",
       "cancel": "Annuler",
       "openWallets": "Ouvrir"
+    },
+    "bitpie": {
+      "warning": "❗ BitPie wallet users: You cannot import the mnemonic phrase directly from your BitPie wallet. <0>Check documentation for details.</0>"
     }
   },
   "createWallet": {
     "newMnemonic": "Générer une nouvelle phrase secrète",
     "thisIsYourPhrase": "Voici votre phrase secrète",
     "instruction": "Sauvegardez votre phrase secrète <1>dans le bon ordre</1> et dans un endroit sécurisé, vous en aurez besoin pour ouvrir votre portefeuille à nouveau.",
     "doNotShare": "Ne partagez jamais votre phrase secrète, l'accès à votre phrase secrète permet d'accéder à votre portefeuille ainsi qu'à vos tokens.",
-    "confirmSaved": "J'ai sauvegardé ma phrase secrète."
+    "confirmSaved": "J'ai sauvegardé ma phrase secrète.",
+    "mnemonicMismatch": "Entered mnemonic does not match."
```

Comment:
is this correct that the script puts english strings into the french json file?

Reply:
wait is this mixed with #538?

Reply:
Ah, crap, I accidentally committed the French translation. Disregard this.

Reply:
Oh, interesting. I didn't push the changes, mega-linter did. It seems that all changes to the tree occurring in the mega-linter's build job is automatically applied. And when I added this translation file check, it is also grabbed and applied. Let me reconsider this...

Reply:
Hi! 
While working on an unrelated PR, English strings were added into the French json upon running `yarn run extract-messages`
If this isn't desired, it can be avoided by adding a default value for keys in the i18n scanner config file, under `options`:
`defaultValue: (lang, namespace, key) => { return '_TRANSLATION_MISSING_' }`

Reply:
@matevz I'll mark this as "changes requested" for now.